sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizeOperationExtensions.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Azure.AI.FormRecognizer.Models
+{
+    /// <summary>
+    /// </summary>
+    public static class RecognizeOperationExtensions
+    {
+        /// <summary>
+        /// </summary>
+        /// <param name="operation"></param>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        public static async ValueTask<Response<IReadOnlyList<RecognizedReceipt>>> WaitForCompletionAsync(this Task<RecognizeReceiptsOperation> operation, CancellationToken cancellationToken = default)
+        {
+            var o = await operation.ConfigureAwait(false);
```

Comment:
From this comment: https://github.com/Azure/azure-sdk-for-net-pr/pull/981/files#r399869464
I thought the idea was to have
```cs
var response = await client.RecognizeReceiptsFromUriAsync(uri, cancellationToken);
```
rather than
```cs
var response = await client.StartRecognizeReceiptsFromUriAsync(uri, cancellationToken).WaitForCompletionAsync(cancellationToken);
```

Reply:
We haven't yet decided to make it look like a non-LRO method... largely because some of these methods can take > 10s, which is a long time to wait, even for an asyc method.  The intention of this change is to still represent it as an LRO (so you start and wait still), but to simplify the machinery around LROs so you can obtain the returned value directly.

Reply:
Will it work if the method has name like `StartAndWaitToRecognizeReceiptsFromUriAsync`?

Reply:
What is the advantage of having it be a single operation rather than an extension on an LRO?

Reply:
It will be better to not change the name of the methods until we have a clear decision on how manage this non-LOR methods.

Reply:
- Less typing and shorter code lines
- No chance to forget about the second `cancellationToken` (not every project has static analyzer that handles it)
- Code with extension method
  ```cs

  var x = client.StartRecognizeReceiptsFromUriAsync(uri, cancellationToken);
  var y = x.WaitForCompletionAsync(cancellationToken).ConfigureAwait(false);
  ```
  looks very similar to the one without extension:
  ```cs
  var x = await client.StartRecognizeReceiptsFromUriAsync(uri, cancellationToken);
  var y = x.WaitForCompletionAsync(cancellationToken).ConfigureAwait(false);
  ```
  It is very easy to forget to add `.ConfigureAwait(false);` for the first method and there will be no compiler exception.

Reply:
> an extension on an LRO?

It isn't on LRO, it is on Task that returns LRO.